src/platform-includes/getting-started-install/kotlin-multiplatform.mdx

```diff
@@ -0,0 +1,48 @@
+### Prerequisites (Android)
+
+Android require disabling auto-init on Android to not clash with the ContentProvider, which auto-initializes the Sentry Android SDK. To do so, add the following to the AndroidManifest.xml file under your androidMain source set:
+
+```kotlin
+<application>
+    <meta-data android:name="io.sentry.auto-init" android:value="false" />
+</application>
+```
+
+To install the Kotlin Multiplatform SDK, you need to add the following to your `build.gradle.kts` file in your shared module:
+
+```kotlin {filename:build.gradle.kt}
+repositories {
+    mavenCentral()
+}
+
+kotlin {
+  val commonMain by getting {
+    dependencies {
+      api("io.sentry:sentry-kotlin-multiplatform:{{ packages.version('sentry.kotlin.kmp', '0.0.1') }}")
```

Comment:
That requires https://github.com/getsentry/sentry-release-registry but it'll fallback to `0.0.1` so lets just be sure to fix the fallback version after being published.